### PR TITLE
twick update pubs 2020

### PR DESCRIPTION
**Pull request by tommeswick**
Worked on my own issue #284 from yesterday and inserted publications myself.

### CODE REVIEW

**Comment by tommeswick** [comment]
Corrected some typos.

File: publications-2020.bib [hunk 1]
```diff
@@ -11,6 +11,87 @@ @book{Wi20_book
...
+
+@Article{JoLaWi20_parallel,
+author = {Jodlbauer, D. and Langer, U. and Wick, T.},
+title = {Parallel Matrix-Free Higher-Order Finite Element Solvers for Phase-Field Fracture Problems},
+journal = {Mathematical and Computational Applications},
+volume = {25},
+number = {3},
+pages = {40},
+year = {2020}
+}
```
Comment (marcfehling) [inline publications-2020.bib:46]: Do you have a `doi`/`url` for this one, too?

**Review by marcfehling — APPROVED** [review]
Only the actual `doi` is necessary for the corresponding `bibtex` flag, so you could technically skip the `https://doi.org/` prefix. But either seems to work, so it's fine.

File: publications-2020.bib (hunk 1 above)
Comment (tommeswick) [reply publications-2020.bib:46]: doi = {https://doi.org/10.3390/mca25030040}

Comment (marcfehling) [reply publications-2020.bib:46]: I mean would you please add the `doi` to your commit :-)

Comment (tommeswick) [reply publications-2020.bib:46]: Done. Added doi into commit.

**Comment by tommeswick** [comment]
Added doi into correct patch.

**Review by jppelteret — APPROVED** [review]
Awesome, thanks a lot!

**Pull request merged**